js/script.js

```diff
@@ -1,16 +1,34 @@
-var url = 'http://api.icndb.com/jokes/random';
-var button = document.getElementById('get-joke');
-button.addEventListener('click', function(){
-  getJoke();
-});
-var paragraph = document.getElementById('joke');
-getJoke();
-function getJoke() {
-  var xhr = new XMLHttpRequest();
-  xhr.open('GET', url);
-  xhr.addEventListener('load', function(){
-    var response = JSON.parse(xhr.response);
-    paragraph.innerHTML = response.value.joke;
-  });
-  xhr.send();
+var prefix = "https://cors-anywhere.herokuapp.com/";
+var tweetLink = "https://twitter.com/intent/tweet?text=";
+var quoteUrl = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
+function getQuote() {
+    $.getJSON(prefix + quoteUrl, createTweet);
+    $.ajaxSetup({ cache: false });
+}
+
+function createTweet(input) {
+    var data = input[0];
```

Comment:
you are taking first element from array input. from where you know that input is an array? check that input contain at least one element before you assign it to variable data